python/pyspark/ml/dl_util.py

```diff
@@ -91,25 +87,20 @@ def create_fn_run_script(
         ----------
         pickled_fn_path: str
             This is the path of the file containing the pickled function, args, and kwargs.
-
         fn_output_path: str
             This is the location where the created file will save the pickled output of
             the function.
-
         script_path: str
             This is the path which will be used for the created pytorch file.
-
         prefix_code: str
             This contains a string that the user can pass in which will be executed before
             the code generated by this class to execute the function and save it. If
             prefix_code is the empty string, nothing will be written before the auto-
             generated code.
-
         suffix_code: str
             This contains a string of code that the user can pass in which will be executed
             after the code generated by this class finishes executing. If suffix_code is
             the empty string, nothing will be written after the auto-generated code.
-
```

Comment:
Would you please add the blank line back between "Parameters" section and "Returns"?